pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "helm";
-  version = "3.8.1";
-  gitCommit = "5cb9af4b1b271d11d7a97a71df3ac337dd94ad37";
+  version = "3.8.2";
+  gitCommit = "6e3701edea09e5d55a8ca2aae03a68917630e91b";
 
   src = fetchFromGitHub {
     owner = "helm";
     repo = "helm";
     rev = "v${version}";
-    sha256 = "sha256-AjNrn46l9gVC7MtGF59QWv+l6qYn+jzopsZtM/2faXY=";
+    sha256 = "sha256-lFAzp7ZxyMZAEO1cNFkEPLgTLEGa6azv36xiTIz4FZY=";
   };
-  vendorSha256 = "sha256-ffbp7J8XDxo/s79pjoiDVbft0pr/lJpuJuKiMpQwkT0=";
+  vendorSha256 = "sha256-FLEydmR+UEZ80VYLxBU1ZdwpdLgTjUpqiMItnt9UuLY=";
 
   doCheck = false;
```

Comment:
Why? Please add a comment and a passthru.tests.version test

Reply:
that change was already pre-existing

Reply:
Yes, that's pre-existing. I'd much rather address that in a different PR though (unless @superherointj beats me to it as they've been sending many PRs recently enabling tests in nix go packages)

Reply:
You should have added here. I only add tests in a separate PR when I don't have any other change in mind. A single commit per package.

Reply:
@superherointj no worries. I was able to cleanly apply that PR to this one.